probe_scraper/transform_probes.py

```diff
@@ -172,19 +170,18 @@ def transform(probe_data, node_data, break_by_channel):
            release channel.
     """
     channels = sorted_node_lists_by_channel(node_data)
-
-    result_data = {}
+    result_data =defaultdict(dict)
```

Comment:
Why did you change this part?

Reply:
I think that default dict is perfect because it won't give key error.